src/public/parts/shells/userSettings/public/index.html

```
@@ -153,6 +153,41 @@ <h2 class="card-title" data-i18n="userSettings.apiKeys.title"></h2>
+						<div class="form-control">
+							<label class="label" for="editorCommandInput"><span class="label-text" data-i18n="userSettings.editorCommand.commandLabel"></span></label>
+							<input type="text" id="editorCommandInput" name="editorCommandInput" class="input input-bordered" required />
+						</div>
+						<div class="form-control">
+							<label class="label" for="editorArgsTemplateInput"><span class="label-text" data-i18n="userSettings.editorCommand.argsTemplateLabel"></span></label>
+							<input type="text" id="editorArgsTemplateInput" name="editorArgsTemplateInput" class="input input-bordered" required />
+							<label class="label py-1"><span class="label-text-alt" data-i18n="userSettings.editorCommand.argsTemplateHint"></span></label>
+						</div>
+						<div class="divider my-1"></div>
+						<div class="grid md:grid-cols-3 gap-2">
+							<input type="text" id="editorTestPath" class="input input-bordered md:col-span-2" data-i18n="userSettings.editorCommand.testPathInput" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)